docs/samples/python/customizing_presidio_analyzer.ipynb

```diff
@@ -0,0 +1,677 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since this recognizer requires the `NlpArtifacts`, we would have to call it as part of the `AnalyzerEngine` flow:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[type: PERSON, start: 0, end: 7, score: 0.85,\n",
+       " type: NUMBER, start: 17, end: 21, score: 0.7,\n",
+       " type: NUMBER, start: 22, end: 24, score: 0.7]"
+      ]
+     },
+     "execution_count": 35,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "text3 = \"Roberto lives in Five 10 Broad st.\"\n",
+    "analyzer = AnalyzerEngine()\n",
+    "analyzer.registry.add_recognizer(new_numbers_recognizer)\n",
+    "\n",
+    "analyzer.analyze(text=text3, language=\"en\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The analyzer was able to pick up both numeric and alphabetical numbers, including other types of PII entities from other recognizers (PERSON in this case)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 4: Calling an external service for PII detection\n",
+    "\n",
+    "In a similar way to example 3, we can write logic to call external services for PII detection. \n",
+    "For a detailed example, see [this part of the documentation](https://microsoft.github.io/presidio/analyzer/adding_recognizers/#creating-a-remote-recognizer).\n",
+    "\n",
+    "[This is a sample implementation of such remote recognizer](https://github.com/microsoft/presidio/blob/main/docs/samples/python/example_remote_recognizer.py).\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 5: Supporting new languages\n",
+    "\n",
+    "Two main parts in Presidio handle the text, and should be adapted if a new language is required:\n",
+    "1. The `NlpEngine` containing the NLP model which performs tokenization, lemmatization, Named Entity Recognition and other NLP tasks.\n",
+    "2. The different PII recognizers (`EntityRecognizer` objects) should be adapted or created."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Adapting the NLP engine\n",
+    "\n",
+    "As its internal NLP engine, Presidio supports both spaCy and Stanza. Make sure you download the required models from spacy/stanza prior to using them. More details [here](https://microsoft.github.io/presidio/analyzer/languages/#configuring-the-nlp-engine). For example, to download the Spanish medium spaCy model: `python -m spacy download es_core_news_md`\n",
+    "\n",
+    "In this example we will configure Presidio to use spaCy as its underlying NLP framework, with NLP models in English and Spanish:"
+   ]
+  },
+  {
```

Comment:
![Screen Shot 2021-03-11 at 13 49 40](https://user-images.githubusercontent.com/1862328/110783152-b9cb7980-8270-11eb-857b-489dc7a8dd82.png)
Not sure if the response should look this way or not...

Reply:
Why not?

Reply:
it's too close to the code looks like a mistake in the typing or something... doesn't look like the response.

Reply:
That's how it's like in jupyter, I don't think I have access over it. Another option is to convert this to markdown but then people can't run it themselves.

Reply:
So ignore and move on :)